spec/services/invoices/add_on_service_spec.rb

```diff
@@ -11,6 +11,8 @@
   let(:applied_add_on) { create(:applied_add_on) }
 
   describe 'create' do
+    around { |test| lago_premium!(&test) }
```

Comment:
What do you think of setting this only when testing the email delivery?

```ruby
describe 'with lago_premium' do
  around { |test| lago_premium!(&test) }

  it 'enqueues an ActionMailer::MailDeliveryJob'
    ...
end
```

That way the premium flag is limited to the smallest unit of tests, and you don't have to turn it off back for the "when license if not premium"